tests/Widget/LinkPagerTest.php

```diff
@@ -83,7 +83,7 @@ public function testDisableCurrentPageButton(): void
         <nav aria-label="Pagination">
         <ul class="pagination justify-content-center mt-4">
         <li class="page-item disabled"><a class="page-link" href="?page=1" data-page="1" aria-disabled="true" tabindex="-1">Previous</a></li>
-        <li class="page-item disabled"><a class="page-link" href="?page=1" data-page="1">1</a></li>
+        <li class="page-item active disabled"><a class="page-link" href="?page=1" data-page="1">1</a></li>
```

Comment:
Why class `active` and `disable` at the same time ?

Reply:
> Why class `active` and `disable` at the same time ?

It styled as **active**, but not clickable bacause of **disabled** (

![2022-03-22 18 28 15 localhost 230c2e292c8e](https://user-images.githubusercontent.com/90403480/159518488-85ef6590-51e0-47dc-958d-35d0df0f9b10.jpg)

Reply:
> I do not understand with only the active class it works well, why add two contradictory classes at the same time?

When - in 

> Work in `yiisoft/demo`:
> 
> ![image](https://user-images.githubusercontent.com/42547589/159520909-a5cc852f-bb3d-427f-bf5c-17c563902e8c.png)

And you can click it and move to current page? But in my case - user restrict click to current page link

Reply:
I understand but why don't you solve it with a custom active class for your case, because it looks weird to have active/disable at the same time.

Reply:
> I understand but why don't you solve it with a custom active class for your case, because it looks weird to have active/disable at the same time.

I need to remove interaction from active link and save highlight it. But in current implemenation i can use only active or disable

Reply:
we can change it no problem

Reply:
> we can change it no problem

It something like replace active link to span as in bootstrap pagination demo